src/main/kotlin/com/example/mykku/auth/tool/GoogleOauthClient.kt

```
@@ -0,0 +1,81 @@
+            restClient.get()
+                .uri(oAuthProperties.userInfoUri)
+                .header(HttpHeaders.AUTHORIZATION, "Bearer $accessToken")
+                .retrieve()
+                .body<GoogleUserInfo>()
+                ?: throw MykkuException(ErrorCode.OAUTH_USER_INFO_FAILED)
+        } catch (e: HttpClientErrorException) {
+            when (e.statusCode.value()) {
+                401 -> throw MykkuException(ErrorCode.OAUTH_INVALID_ACCESS_TOKEN)
+                else -> throw MykkuException(ErrorCode.OAUTH_USER_INFO_FAILED)
+            }
+        } catch (e: HttpServerErrorException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)